tests/rules/tcp-mss/test.rules

```diff
@@ -0,0 +1,4 @@
+alert tcp any any -> any any (msg:"Testing mss"; tcp.mss:50; sid:1;)
+alert tcp any any -> any any (msg:"Testing mss"; tcp.mss:>123; sid:2;)
+alert tcp any any -> any any (msg:"Testing mss"; tcp.mss:<536; sid:3;)
+alert tcp any any -> any any (msg:"Testing mss"; tcp.mss:123-456; sid:4;)
```

Comment:
> Wondering if it would make sense to include other rules with the other possible operands here...

I wrote the rules based off the format instruction for tcp.mss provided on the userguide.

So I don't know exactly.

But if you say to add it, then no problem still.

Reply:
That's why I said I'm wondering. On the one hand, it makes sense to showcase what the userguide says. On the other, it can be good to stretch things a bit and see how Suri reacts, you see?

Reply:
> That's why I said I'm wondering. On the one hand, it makes sense to showcase what the userguide says. On the other, it can be good to stretch things a bit and see how Suri reacts, you see?

Yes! I understand.